src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -9,6 +9,7 @@
 */
 public class TorpedoStore {
 
+	private Random generator;
```

Comment:
A tabulátor használata a kód stílusa szempontjából nem helyes, valamint én inkább egyből inicializáltam is volna itt a változót a jobb átláthatóság végett.